invokeai/app/invocations/image.py

```diff
@@ -32,16 +34,14 @@ class ImageOutput(BaseInvocationOutput):
     # fmt: off
     type: Literal["image"] = "image"
     image:      ImageField = Field(default=None, description="The output image")
+    width:   Optional[int] = Field(default=None, description="The width of the image in pixels")
+    height:  Optional[int] = Field(default=None, description="The height of the image in pixels")
+    mode:    Optional[str] = Field(default=None, description="The image mode (ie pixel format)")
```

Comment:
Is this useful for passing between nodes at all? I'm presumably in the image already, and may not have a ton of meeting as just a string.

Reply:
I suppose not, deleted